app/forms/support_interface/application_forms/pick_course_form.rb

```diff
@@ -54,11 +56,11 @@ def courses
         Course
           .current_cycle
           .includes(course_options: [:site])
-          .where(code: sanitize(course_code))
+          .where(code: sanitize(course_code), open_on_apply: true, exposed_in_find: true)
```

Comment:
can we use the [scopes](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/models/course.rb#L13) `exposed_in_find` and `open_on_apply` instead?